app/concerns/authentication/api.rb

```diff
@@ -26,7 +26,7 @@ def decoded_api_token
 
     # handle edge case errors more reasonably
     def decode_api_token
-      auth_header = request.headers['Authorization'].to_s.split(' ').last
+      auth_header = request.headers['Authorization'].to_s.split.last
```

Comment:
Which lint affects this? I don't like this change much – it looks like [there's some fallback to a `$;` global when this isn't specified](https://ruby-doc.org/core-3.0.1/String.html#method-i-split), and it seems a little magic to rely on this being a single whitespace, but maybe I'm just being a bit weird :P

Reply:
[Style/RedundantArgument](https://docs.rubocop.org/rubocop/cops_style.html#styleredundantargument), apparently